packages/website/src/snippets/Components/Actions.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from "react";
+import { Actions } from "..";
+import { formatMessage } from "../formatMessage";
+
+export default function ActionsExample() {
+  return (
+    <Actions
+      size="medium"
+      primaryAction={{
+        label: formatMessage("Ok"),
```

Comment:
No need for formatMessage anymore, I would probably remove it in the examples

Reply:
I was unsure, but maybe you're right... better to keep it simple.